apps/kyb-app/src/components/atoms/Stepper/components/atoms/Breadcrumbs/Breadcrumbs.Label.tsx

```diff
@@ -1,19 +1,12 @@
 import { BaseLabel } from '@/components/atoms/Stepper/components/atoms/BaseLabel/BaseLabel';
 import { BreadcrumbsLabelProps } from '@/components/atoms/Stepper/components/atoms/Breadcrumbs/types';
-import { StepperLabelsMap } from '@/components/atoms/Stepper/types';
-
-export const baseLabelsMap: StepperLabelsMap = {
-  idle: ({ text }) => <BaseLabel variant="idle" text={text} />,
-  warning: ({ text }) => <BaseLabel variant="warning" text={text} />,
-  completed: ({ text }) => <BaseLabel variant="completed" text={text} />,
-};
+import { ctw } from '@ballerine/ui';
 
 export const Label = ({ active, text, state }: BreadcrumbsLabelProps) => {
-  const LabelComponent = baseLabelsMap[state] || null;
-
-  return active && state === 'idle' ? (
-    <BaseLabel variant="current" text={text} />
-  ) : LabelComponent ? (
-    <LabelComponent text={text} />
-  ) : null;
+  return (
+    <BaseLabel
+      className={ctw({ 'font-bold': active, 'opacity-50': state === 'completed' })}
+      text={text}
+    />
+  );
 };
```

Comment:
Creating a component for two CSS classes seems like overkill. Can't these classes be passed from outside?